12_calculator/solution/calculator-solution.js

```diff
@@ -15,7 +15,7 @@ const multiply = function (array) {
 };
 
 const power = function (a, b) {
-  return Math.pow(a, b);
+  return a**b;
```

Comment:
Nit for consistency of formatting
```suggestion
  return a ** b;
```